.github/workflows/build.yml

```diff
@@ -16,6 +27,8 @@ jobs:
         target: [ Gplay, Fdroid ]
     steps:
       - uses: actions/checkout@v2
+      - name: Generate CI gradle.properties
+        run: echo "$CI_GRADLE_PROPERTIES" > gradle.properties
```

Comment:
Use `>>` instead of `>` to append line in this file?

Reply:
Yes we can do that, but the point was to be decoupled. So the whole configuration for each flow is unique with no dependencies. However, I can change it with append in order to reduce the amount of code lines

Reply:
in addition with decoupled, if a developer change something (accidentally or not in gradle.properties) it will not interfere with our github action, while its sensitive with 7 GB ram. what do you think?

Reply:
It's not ideal, if developer add a required config in gradle.properties (for instance something similar to `android.useAndroidX`, it will be stripped by the CI if the developer forget to update the workflow. So I prefer to use `>>` or maybe better command line argument.

Also gradle.properties file should be well documented.

Reviewers take particular attention to changed in those files anyway.

Reply:
Yes, it might be better from that point of view. I rebased the branch and changed the implementation with  command line arguments